README.md

```diff
@@ -1,2 +1,49 @@
 # in-toto
 in-toto is a series of scripts to protect the supply chain integrity.
+
+# About in-toto
+in-toto guarantees that the end-user (or owner) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (participants) have perfomed the specified tasks and there haven't been any malicious changes in the files.
+
+in-toto requires a project layout that specifies the functionaries and the tasks they are supposed to perform.
+After each functionary performs its task, a link metadata is generated.
+This metadata is used to verify the intermediate and final products with the project layout.
+
+# in-toto demo
+You can try in-toto by running the demo application at [https://github.com/in-toto/in-toto/tree/develop/demo].
+The demo basically outlines three users viz., Alice (project owner), Bob (functionary) and Carl (functionary) and how in-toto helps to specifiy a project layout and verify that the layout has been followed in a correct manner.
+
+- Alice speficies the layout with Bob and Carl's public keys and signs it with her private key.
+- Bob writes a python script and in-toto hashes the product and stores it to a piece of link metadata. The metadata is signed with Bob's private key. Bob sends the python script to Carl for packaging.
+- Carl packages the script into a tarball and in-toto stores the hash to the metadata.
+- The client (end-user) can now verify the final product which contains the layout, tarball, the both link metadata. The in-toto-verify command basically checks whether the layout hasn't expried and if the steps were performed according to the layout definitions.
+
+# Using in-toto
+* Clone our [repository](https://github.com/in-toto/in-toto.git).
+* Run setup.py
+* Start using the in-toto run and verify scripts
```

Comment:
Maybe you could call this section something like "Getting started" with the following sub sections:

1. **Installation**
similar to what you wrote in `# Contributing to in-toto`
2. **Create layout**
This will be rather complex. Maybe you can start with a simple description of the parts a layout must/should contain (steps, inspections, functionary keys) and the basic matchrule grammar?
3. **Perform software supply chain steps**
A brief description of the command line tools, `in-toto-run` and `in-toto-record`. What they do and how to use them.
4. **Release final product**
We do not strictly specify how in-toto metadata files should be shipped out together with the final product. For now this section should mention something like:
*In order to verify the final product with in-toto, the verifier must have access to the `layout`, the `*.link` files and the project owner's `public key`.*
5. **Verify final product**
A brief description of the command line tool, `in-toto-verify`.